ngdo-pro/WCS_Projet3
# Sogenactif bundle

Hi team,

Please, remember to : 

- composer install (new parameters are created, you will need to adapt them to your project)
- chmod 755 src/SogenactifBundle/Api/bin/* (request and response files from Sogenactif Api)
- change absolute paths in /src/Sogenactif/Api/params/pathfile to fit your project 
- app/console doctrine:migrations:migrate (you may need to reset your database)
- app/console doctrine:fixture:load

To test it : 
- Login as user
- Search 
- Select a baptism
- Confirm
- Click on a payment mean
- Enter a fake card number like 1234567812345600 ( it has to end with 00 to be a valid payment, 05 to fail )
- Return to website with api button (Retour à la boutique)

Autoresponse does work online, can't be tested offline
